sdk/communication/Azure.Communication.Administration/tests/Azure.Communication.Administration.Tests.csproj

```diff
@@ -18,6 +18,8 @@
   <!-- Shared source from Azure.Core -->
   <ItemGroup>
     <Compile Include="$(AzureCoreSharedSources)ConnectionString.cs" Link="Shared\%(RecursiveDir)\%(Filename)%(Extension)" />
+    <Compile Include="..\..\Shared\src\CommunicationRecordedTestSanitizer.cs" Link="Shared\Communication\CommunicationRecordedTestSanitizer.cs" />
```

Comment:
We could put this in the `Azure.Communication.Common.Tests` project and then add reference. Having shared internal types are mainly important for public libraries.

Reply:
Moved to common.tests. Thanks for the clarification!